java/tsfile/src/main/java/org/apache/tsfile/read/reader/page/AlignedPageReader.java

```diff
@@ -92,6 +94,38 @@ public AlignedPageReader(
     this.valueCount = valuePageReaderList.size();
   }
 
+  @TestOnly
+  public AlignedPageReader(
+      PageHeader timePageHeader,
+      ByteBuffer timePageData,
+      Decoder timeDecoder,
+      List<PageHeader> valuePageHeaderList,
+      List<ByteBuffer> valuePageDataList,
+      List<TSDataType> valueDataTypeList,
+      List<Decoder> valueDecoderList,
+      Filter globalTimeFilter,
+      boolean isLazyLoaded) {
```

Comment:
what's this parameter for? we shouldn't change the interface method signature

Reply:
I changed to LazyLoadPageData[].